### PR TITLE
TACO-427 Add cmp tracking key val

### PR DESCRIPTION
**Pull request by szymon-nawrocki**
### Ticket
https://fandom.atlassian.net/browse/TACO-427

### Description
In this PR I've added `cmp` targeting key value that is sent along GAM requests.
I've also added sending information about CMP to `adengadinfo` data table. Because it is temporary, I've added it to `rollout_tracking` column.

### CODE REVIEW

File: @types/window/index.d.ts [hunk 1]
```diff
@@ -67,4 +67,5 @@ interface Window {
 			getVariationMap: () => Record<string, { id: string; name: string }>;
 		};
 	};
+	OneTrust?: any;
```
Comment (domjanik) [inline @types/window/index.d.ts:70]: Maybe we could type this object a bit? At least include the most important methods/values here?

File: src/ad-services/identity-setup/index.ts [hunk 2]
```diff
@@ -68,4 +71,8 @@ export class IdentitySetup implements DiProcess {
 			}
 		});
 	}
+
+	private getCmp(): '1t' | 'toi' {
```
Comment (domjanik) [inline src/ad-services/identity-setup/index.ts:75]: Maybe we could extract '1t' | 'toi' to separate type / enum? Like:
type CmpTypes = '1t' | 'toi' 
or 
enum CmpTypes {
  OneTrust = '1t',
  TrackingOptIn = 'toi'
}

Comment (szymon-nawrocki) [reply src/ad-services/identity-setup/index.ts]: Great idea, changed ✅

File: src/ad-tracking/compilers/slot-tracking-compiler.ts [hunk 3]
```diff
@@ -23,6 +23,10 @@ function checkOptOutSale(): string {
 	return '';
 }
 
+function getCmp(): '1t' | 'toi' {
```
Comment (domjanik) [inline src/ad-tracking/compilers/slot-tracking-compiler.ts:26]: You can re-use CMP_TYPE from identity-setup in here to preserve consistency.

**Review by domjanik — CHANGES_REQUESTED** [review]

**Review by domjanik — DISMISSED** [review]

**Review by domjanik — APPROVED** [review]

**Pull request merged**